batch_convert.qml

```diff
@@ -340,14 +340,22 @@ MuseScore {
           } // RowLayout
         CheckBox {
           id: exportExcerpts
-          text: /*qsTr("Export linked parts")*/ qsTranslate("action", "Export parts")
+          text: /*qsTr("Export linked parts")*/ qsTranslate("action", "Export Parts")
```

Comment:
I had to revert this change, this string gets 'recycled' from MuseScore, so need to follow the spelling there in order to get translated properly